ipns/pb/ipns-record.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package github.com.boxo.ipns.pb;
 
-option go_package = ".;pb";
```

Comment:
Based off my reading around, the usage of a `.` for the package name is essentially a no-op and is not recommended. https://github.com/golang/protobuf/issues/1249#issuecomment-736859185

Reply:
IIUC `.` tells protoc to generate the file in the same directory, and `pb` indicates the go package name